### ISSUE TITLE
Allow passing custom plotting options to echogram

### ISSUE THREAD

**Issue by leewujung** [OSOceanAcoustics/echoshader#106]
Currently the plot options such as `width` and `height` are fixed as they are initiated in init as `self.gram_opts`. We need to preserve the way holoviews plots are customized and enable users to do something like:
```python
ds_MVBS.eshader.echogram_multiple_frequency(layout="composite").opts(
    "width": 1200, "height": 400, ...
)
```

**Comment by ldr426** [comment]
Please directly change the property of Echogram class:
```python
self.gram_opts = {
    "Image": {
        "cmap": "jet",
        "colorbar": True,
        "tools": ["box_select", "lasso_select", "hover"],
        "invert_yaxis": False,
        "width": 600,
        },

    "RGB": {"tools": ["hover"], 
            "invert_yaxis": False, 
            "width": 600
            },
}
```

**Comment by leewujung** [comment]
I did this on the cruise just so that I can change the sizes, but the users should not have to touch the underlying code. We need to support the use case as I listed above.

**Comment by valentina-s** [comment]
This is now implemented in version 0.1.0.
